eth_defi/uniswap_v3/swap.py

```diff
@@ -76,6 +77,14 @@ def swap_with_slippage_protection(
 
     if max_slippage == 0:
         warnings.warn("max_slippage is set to 0, this can potentially lead to reverted transaction. It's recommended to set use default max_slippage instead (0.1 bps) to ensure successful transaction")
+    
+    if type(pool_fees) == int:
+        pool_fees = [pool_fees]
+    elif type(pool_fees) != list:
+        raise ValueError("pool_fees must be passed as an int or int list. Each fee is BPS x 100")
+    
+    for fee in pool_fees:
+        assert (fee > 1 and type(fee) == int), "pool_fees must be passed as an int or int list. Each fee is BPS x 100"
```

Comment:
You can actually check if the fee is in the fee enum here, also raise ValueError instead of assert would be nice